src/inference/src/ie_core.cpp

```diff
@@ -533,7 +533,8 @@ class CoreImpl : public ie::ICore, public std::enable_shared_from_this<ie::ICore
             if (batch_mode != config.end()) {
                 const auto disabled = batch_mode->second == CONFIG_VALUE(NO);
                 // no need for this config key in the rest of loading
```

Comment:
maybe pls add comment that virtual plugins like AUTO/MULTI need the config to e.g. deduce the #requests correctly

Reply:
yes, updated the comment